src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -296,15 +299,26 @@ private static void assertGVCFHasOnlyOneSample(final String variantPath, final V
         }
     }
 
-    @VisibleForTesting
-    static SortedMap<String, Path> loadSampleNameMapFile(final Path sampleToFileMapPath) {
+    /**
+     * load a tab delimited new line separated file of sample name to URI mapping:
```

Comment:
Add note that this version preserves the original ordering of the sample map file, and should not be called directly by this tool (call `loadSampleNameMapFileInSortedOrder()` instead)

Reply:
noted